### PR TITLE
fix #12

### LINKED ISSUE #12: Certificate generation does not work.
Currently, we can not customize the certificate.

### PR DESCRIPTION
**Pull request by niccokunzmann**
This changes some paths so the javascript files are included again

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@Alanh168, would you like to merge this?

File: js/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ function setDownloadURI() {
 function updateImage() {
   var query_string = generateQueryString();
   var preview = document.getElementById("preview");
-  preview.src = "../img/coderdojocertificate.svg" + query_string;
```
Comment (niccokunzmann) [inline js/index.js:74]: Javascript is executed relative to the html file is is in. You can include many JS files from many places and they all use your html file location. @Alanh168

**Review by Alanh168 — APPROVED** [review]
All paths check out

File: js/index.js (hunk 1 above)
Comment (Alanh168) [reply js/index.js:74]: Oh I thought Javascript executes relative to where the JS file is located. Now I understand, thanks

**Comment by niccokunzmann** [comment]
@Alanh168 If you like to contribute more, please create a pull-request whenever you like.

**Comment by Alanh168** [comment]
Will do! I'll be going over the code today and tonight so I know exactly how everything works. I'm not too familiar with how to use Javascript to create customizable svg images so it should be educational.

**Pull request merged**